src/main/kotlin/org/acejump/session/Session.kt

```diff
@@ -124,7 +125,7 @@ class Session(private val editor: Editor) {
       is TaggingResult.Jump -> {
         tagJumper.jump(result.offset, shiftMode)
         tagCanvas.removeMarkers()
-        end()
+        end(true)
```

Comment:
Maybe this can change this line to return the `queryText` and `tag` selected:

https://github.com/acejump/AceJump/blob/2c0a236ed84df23b5ab353657fe251157b567d9f/src/main/kotlin/org/acejump/search/Tagger.kt#L56

Then add the query and tag as parameters to `TaggingResult.Jump`, something like this:

```kotlin
if (entry solves queryText)
   TaggingResult.Jump(query = queryText.substringBefore(entry.key), tag = entry.key, offset = entry.value)
```